src/main/java/org/folio/qm/domain/entity/RecordCreationStatusUpdate.java

```diff
@@ -3,12 +3,10 @@
 import java.util.UUID;
 
 import lombok.Builder;
-import lombok.ToString;
 import lombok.Value;
 
 @Value
 @Builder
-@ToString
```

Comment:
As I remember it is needed for log purpose

Reply:
![image](https://user-images.githubusercontent.com/37537790/109645634-cc063300-7b5f-11eb-945f-e45163ddb3bf.png)
form the documentation - @Value includes @ToString

Reply:
👍